### PR TITLE
Final UI Updates to Website

### PR DESCRIPTION
**Pull request by piperlincoln**
Finished final UI updates to website. Should now have consistent theme across all elements. Some particularly important updates are listed below.

- Wrapped collection and item boxes so they don't overflow out of the right side of the container.
- Moved navigation menu bar to left column.
- Reformatted collection boxes on "Browse" page so viewer does not have option to edit.
- Reformatted favorite icon in item boxes.

The images of the final design of each website page can be seen [here](https://docs.google.com/presentation/d/1m4ZadUFLy-VrZXSNnlECoSHgau_GROV5HoqScsPPJQM/edit#slide=id.gbdf31cd5da_0_0).

### CODE REVIEW

File: src/components/BrowsePageComponent.tsx [hunk 1]
```diff
@@ -1,20 +1,22 @@
+import { Auth } from "aws-amplify";
 import React, { useCallback, useEffect } from "react";
 import { useContext } from "react";
 import CollectionStore from "../Database/CollectionStore";
```
Comment (firicano33) [inline src/components/BrowsePageComponent.tsx:4]: I see that the collection is being called from the database

File: src/components/BrowsePageComponent.tsx [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
 import { CollectionModel } from "../domain/CollectionModel";
 import { useState } from "react";
 import "./PageLayoutComponent.css";
-import { CollectionBox } from "./CollectionBox";
+import { FakeCollectionBox } from "./FakeCollectionBox";
 import CollectRImage from "../images/CollectR.jpg";
 import { Link } from "react-router-dom";
 import FakeCollectionStore from "./FakeCollectionStore";
```
Comment (firicano33) [inline src/components/BrowsePageComponent.tsx:11]: I see there is a FakeCollection.  What is that being used for?

**Review by firicano33 — APPROVED** [review]
Looks great.  Good job!

File: src/components/BrowsePageComponent.tsx (hunk 2 above)
Comment (piperlincoln) [reply src/components/BrowsePageComponent.tsx:11]: A better word would be "mock" but those are the collections displayed on the "Browse" page as a sample of what the user could search for!

File: src/components/FakeCollectionBox.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
```
Comment (Taracuda) [inline src/components/FakeCollectionBox.tsx:1]: Thank you for creating the fake collection box. This makes complete sense and I can see how this keeps viewers from being able to make changes to collections when they browse.

**Review by Taracuda — APPROVED** [review]

File: src/components/FakeCollectionBox.tsx (hunk 3 above)
Comment (piperlincoln) [reply src/components/FakeCollectionBox.tsx:1]: You're very welcome! I'm glad it turned out alright!

**Pull request merged**